cmd/manager/main.go

```diff
@@ -101,6 +105,7 @@ func main() {
 	flag.StringVar(&cachePath, "cache-path", "/var/cache", "The local directory path used for filesystem based caching")
 	flag.BoolVar(&operatorControllerVersion, "version", false, "Prints operator-controller version information")
 	flag.StringVar(&systemNamespace, "system-namespace", "", "Configures the namespace that gets used to deploy system resources.")
+	flag.StringVar(&globalPullSecret, "global-pull-secret", "", "The namespace/name of the global pull secret that is going to be used to pull bundle images.")
```

Comment:
Just as a thing to note, there is a [`flag.Var()` method](https://pkg.go.dev/flag#FlagSet.Var) that would allow us to implement a custom flag value with custom logic for handling how it is set, including validations. 

I did something similar to this in kapp back when we were contributing preflight checks: https://github.com/carvel-dev/kapp/blob/ca6887d26d782636fe62a2b3f3bbbf5c91a965f3/pkg/kapp/preflight/registry.go#L64-L97

Might be overkill for this particular case, but this may be a good way to handle any custom flag validations we want in the future